server/game/gameSystems/DelayedEffectSystem.ts

```diff
@@ -24,10 +28,12 @@ export class DelayedEffectSystem<TContext extends AbilityContext = AbilityContex
     public override readonly effectDescription: string = 'apply a delayed effect';
 
     protected override defaultProperties: IDelayedEffectSystemProperties = {
+        title: '',
```

Comment:
Might be better to just put it as `null` since that's what we typically check for as the 'empty value'